Objects/longobject.c

```diff
@@ -6775,3 +6776,115 @@ int PyLong_AsUInt64(PyObject *obj, uint64_t *value)
 {
     LONG_TO_UINT(obj, value, "C uint64_t");
 }
+
+
+static const PyLongLayout PyLong_LAYOUT = {
+    .bits_per_digit = PyLong_SHIFT,
+    .digits_order = -1,  // least significant first
+    .digit_endianness = PY_LITTLE_ENDIAN ? -1 : 1,
+    .digit_size = sizeof(digit),
+};
+
+
+const PyLongLayout*
+PyLong_GetNativeLayout(void)
+{
+    return &PyLong_LAYOUT;
+}
+
+
+int
+PyLong_Export(PyObject *obj, PyLongExport *export_long)
+{
+    if (!PyLong_Check(obj)) {
```

Comment:
Depending on this check, the tests should allow for int subclasses (I assumed that we only allowed exact ints, but that doesn't seem the case?)

Reply:
> I assumed that we only allowed exact ints

Why?  That works in the same way as the rest of the PyLong_* C-API.

Reply:
Because the tests didn't test for int subclasses I wasn't sure (I don't use much the PyLong_* C API actually)

Reply:
I added a test on int subclass.